config_zh_cn.yml

```diff
@@ -0,0 +1,14110 @@
+#######################################################################
+#                          PetBlocks                                  #
+# 		            	   by Shynixn							      #
+#######################################################################
+
+# You can get started and find the permissions at the official documentation.
+# https://shynixn.github.io/PetBlocks
+
+############################
+
+# Data-collecting settings
+
+# These are settings to send anonymous stats to https://bstats.org/plugin/bukkit/PetBlocks.
+# As this does not have an impact on your PetBlocks experience at all I would like you to set it on true as knowing that my
+# plugin gets used by many server helps me focusing on new updates ;)
+
+############################
+
+metrics: true
+
+############################
+
+# Config version
+
+# Change the config-version when you have read the update log and see a change of
+# the config.yml. The config.yml does not change every update and the plugin automatically
+# checks if your config-version is compatible with the installed plugin.jar.
+
+############################
+
+config-version: 1
+
+############################
+
+# Database settings
+
+# All pets are stored in a SQLite database per default which is basically the
+# PetBlocks.db file in the plugin folder.
+# Available types are 'sqlite' and 'mysql'.
+# Use the same mysql database for 2 or more server in order to allow cross server pets.
+
+############################
+
+sql:
+  type: 'sqlite'
+  host: localhost
+  port: 3306
+  database: ''
+  usessl: false
+  table-prefix: 'SHY'
+  username: ''
+  password: ''
+
+############################
+
+# Command settings
+
+# There are commands available in PetBlocks you can customize by yourself.
+# This means you can disable the command entirely, change the name, description
+# and even permission.
+
+############################
+
+commands:
+  petblock:
+    enabled: true
+    command: petblock
+    useage: /<command>
+    description: Opens the pet GUI.
+    permission: petblocks.command.use
+    permission-message: You don't have permission
```

Comment:
Please also translate the description and permission-message.

Reply:
Leave useage message and permission be. Only description and permission - message needs a translation. Thanks :)